### PR TITLE
[+]Fixed problems and the csv file imported can be revealed in the screen!

### PR DESCRIPTION
**Pull request by eevadai**
Further Consideration:
Violates Clean Architecture!

### CODE REVIEW

**Comment by ScottCTD** [comment]
You can add a presenter to output the entries that are imported, just like the bill update use case and the user join use case.

**Review by ScottCTD — APPROVED** [review]
Looks good!

It is better to add support for csv header. 
But for now, it is okay.

File: src/main/java/billgates/view/gui/TopMenuBar.java [hunk 1]
```diff
@@ -126,20 +138,41 @@ public JMenuItem getFileMenu() {
...
+            BufferedReader br = new BufferedReader(new FileReader(file));
+            String line;
+            while ((line = br.readLine()) != null) {
+                String[] values = line.split(",");
+                csvFile.add(Arrays.asList(values));
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        System.out.println(csvFile);
```
Comment (Lei-Tin) [inline src/main/java/billgates/view/gui/TopMenuBar.java:174]: Do we want to print this in the actual console?

**Review by Lei-Tin — APPROVED** [review]
Looking good! Just remember to commit and remove the print line so that our code is cleaner before merging!

**Review by Lei-Tin — APPROVED** [review]
GOOD!

**Review by ScottCTD — APPROVED** [review]
GOOD

**Pull request merged**